java/code/src/com/redhat/rhn/domain/recurringactions/RecurringActionFactory.java

```diff
@@ -86,24 +88,31 @@
      * @return the list of {@link RecurringAction}s
      */
     public static List<? extends RecurringAction> listAllRecurringActions(User user) {
-        Org org = user.getOrg();
+        List<Org> org;
```

Comment:
This commit looks very good from the technical POV (apart of the variable naming: when we have a `List` of `Org`, the variable should be called `orgs`), but I think we *shouldn't* display everything for the SAT admin:

1. in the list of all systems, SAT admin can't see systems of other organizations (only his/her own)
2. same goes for the system groups (they can only see groups in their org)

I think displaying recurring actions for servers/groups that the user can't see would be confusing and therefore I suggest reverting this commit.

Reply:
Adapted to just show other `Organizations` schedules.